Update/kage_002.txt

```diff
@@ -4170,19 +4170,19 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a4856e>まどか</color>", NULL, "<color=#a4856e>Madoka</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200037", 256, TRUE);
 	OutputLine(NULL, "（さっさと署に帰って、早引けするか仮眠室で横になりたいっての）",
-			NULL, "(I should go back to the station and head to the break room right away.)", Line_Normal);
+			NULL, "(I want to go back to the station and head to the break room right away.)", Line_Normal);
 	ClearMessage();
 
 //rまして、こんなヘロヘロ状態で『あの人』と喧嘩の仲直りしろ、なんて冗談を通り越してむしろ嫌がらせにしか思えない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "まして、こんなヘロヘロ状態で『あの人』と喧嘩の仲直りしろ、なんて冗談を通り越してむしろ嫌がらせにしか思えない。",
-			NULL, "Still, <i>she</i> would harass her the moment she walked by, and it would be really difficult to make up with her after their fight in this totally exhausted state.", Line_Normal);
+			NULL, "Besides, being told to make up with <i>her</i> in this totally exhausted state went beyond a joke and sounded like nothing but harrassment to Madoka.", Line_Normal);
 	ClearMessage();
 
 //r口を開いたところで不機嫌な気分のまま、素直に謝罪なんて出るわけもない。せいぜい、皮肉か嫌味が出てくるのが関の山で、余計にこじれるのが目に見えている…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "口を開いたところで不機嫌な気分のまま、素直に謝罪なんて出るわけもない。せいぜい、皮肉か嫌味が出てくるのが関の山で、余計にこじれるのが目に見えている…。",
-			NULL, "She would be in a bad mood the moment she opened her mouth, and it would be difficult to get an apology out. She'd do her best to keep the sarcasm and hate from coming out, but it would still be readily apparent...", Line_Normal);
+			NULL, "She would be in a bad mood the moment she opened her mouth, and it would be difficult to get an apology out. It was readily apparent that she'd just fire barbs and sarcasm, only souring things more...", Line_Normal);
```

Comment:
```suggestion
			NULL, "She would be in a bad mood the moment she opened her mouth, and it would be difficult to get an apology out. It was readily apparent that she'd just fire barbs and sarcasm, souring things even more...", Line_Normal);
```